src/css/style.css

```diff
@@ -0,0 +1,151 @@
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  background: #191765;
+}
+
+#firstBackground {
+  background-image: url("../img/two_circles_header.png");
+  background-repeat: no-repeat;
+}
+
+.nav-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 10px solid goldenrod;
+}
+
+.ham-button {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  color: white;
+  background-color: #191765;
+  font-size: 25px;
+  margin-left: auto;
+  cursor: pointer;
+  border: 0;
+}
+
+.justifyFlexcomponentsXY {
+  justify-content: center;
+  align-items: center;
+}
+
+hr.solid {
+  width: 100%;
+  justify-content: center;
+  border: 1px solid #646369;
+  top: calc(50% - 0px / 2 + 0.5px);
+}
+
+.knowledge {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  flex-wrap: wrap;
+}
+
+.skillbox {
+  width: 50%;
+}
+
+/* Navbar styles */
+.navbar {
+  color: #28352f;
+  font-weight: 600;
+  font-size: 20px;
+  display: flex;
+  align-items: center;
+  max-height: 45px;
+  padding: 10px 24px;
+}
+
+.shown-mobile > nav {
+  display: none;
+}
+
+/* Introduction style */
+.intro {
+  display: flex;
+  flex-direction: column;
+  padding: 64px 24px;
+  margin: 24px;
+  background-image: url("../img/Top_circles_mobile.png");
+  background-size: cover;
+}
+
+/* Styles of the Figma templates */
+.titleHeader {
+  color: #7d73fc;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 48px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  margin: 12px;
+}
+
+.subtitleHeader {
```

Comment:
- [ ] In following with the design template the subtitle header text should be bolder than it is. Please fix it.